packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/tests/rich-text-mention-users-view.stories.ts

```diff
@@ -0,0 +1,67 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../../../../utilities/tests/storybook';
+import { richTextMentionUsersViewTag } from '..';
+import {
+    bodyFont,
+    bodyFontColor
+} from '../../../../../theme-provider/design-tokens';
+
+interface RichTextMentionUsersViewArgs {
+    mentionHref: string;
+    mentionLabel: string;
+}
+
+const richTextMentionUsersViewDescription = 'The rich text mention users view component is used to render the at(@) mentioned users in the nimble rich text editor and nimble rich text viewer.';
+
+const metadata: Meta<RichTextMentionUsersViewArgs> = {
+    title: 'Incubating/Rich Text View: Mention Users Node',
```

Comment:
This is not a user facing component right? End users will not construct these directly. This is just an implementation detail. If so move it under Tests/

```suggestion
    title: 'Tests/Rich Text Mention: Users',
```

Reply:
Yes, I have moved it to Tests/ as suggested